content/articles/getting-started-clickfunnels.markdown

```diff
@@ -0,0 +1,222 @@
+| Email                  | Make sure this is accurate. You're going to be sent emails verifying your information that you will be required to approve. If you enter your email address incorrectly, this email will not be delivered and you could face a suspension of your domain until it is fixed. |
+| Organization Name      | If the domain is to be registered for a company, you'll enter that here. For now, let's leave this blank.                                                                                                                                                                   |
+| Job Title              | As above, if the domain were being registered for a company, you'll enter your job title. Let's leave this blank too.                                                                                                                                                       |
+| Phone                  | Enter your complete phone number starting with the country code.                                                                                                                                                                                                            |
+| Fax                    | This is usually relevant for company registrations—we'll leave this blank as well.                                                                                                                                                                                           |
+| Address                | Enter your complete physical or postal address.                                                                                                                                                                                                                             |
+
+
+## Step 4: Understanding the domain homepage
+
+Let's go through what you're seeing on the home page for the domain to make it a little less daunting.
+
+### Domain Delegation
+
+What you see in this card are the name servers. Name servers work as a directory that translate domain names into IP addresses and are important because they make things easy to find across the Internet.
+
+<div class="mb4 aspect-ratio aspect-ratio--16x9 z-0">
+  <iframe src="https://www.youtube.com/embed/2WdF1zT01HY" class="aspect-ratio--object" frameborder="0" allow="accelero    meter; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+
+You don't need to change the name servers when you move your domain from ClickFunnels.
+
+The only reason you would need to change the name servers is if you are looking to [manage your DNS with another provider](/articles/setting-name-servers/#pointing-the-name-servers-to-another-provider), like GoDaddy or CloudFlare, and keep your registration with us.
```

Comment:
Should we put a caveat about the 60 day hold here?

Reply:
@SarahCaminiti1 i'm not sure it makes sense to mention here when we're speaking about delegation rather than anything that updates the registrant info.